tests/admin_filters/tests.py

```diff
@@ -737,7 +792,7 @@ def test_simplelistfilter(self):
         choices = list(filterspec.choices(changelist))
         self.assertEqual(choices[0]['display'], 'All')
         self.assertEqual(choices[0]['selected'], True)
-        self.assertEqual(choices[0]['query_string'], '?')
+        self.assertEqual(choices[0]['query_string'], '?publication-decade={0}'.format(sentinel))
```

Comment:
Hmm no, does not work because then the sentinel is None, which might be a valid value too...